src/rpc/blockchain.cpp

```diff
@@ -2184,6 +2206,7 @@ static const CRPCCommand commands[] =
     { "blockchain",         "preciousblock",          &preciousblock,          true,  {"blockhash"} },
 
     /* Not shown in help */
+    { "hidden",             "getbestchainlock",       &getbestchainlock,       true,  {} },
```

Comment:
is it really important that this is hidden?

Reply:
I didn't want to expose it because it's "the best chainlock" but not "the best chainlock with a known block" and I'm afraid it could cause some confusion. 976447c should fix this.

Reply:
would it be better to augment the function to only ever consider a CLSIG "best" if the block is known locally? If the block is not known locally I would think the CLSIG would be saved locally but not used until it's block comes in

Reply:
The way it works internally: there is a "best chainlock" and a "best chainlock with known block". This internal logic doesn't have to be exposed ofc, we can change the name from `getbestchainlock` to smth that causes less confusion probably. I'm open for suggestions :)